### DISCUSSION TITLE
Is there a way to get a users pins with the GitHub REST API?

### DISCUSSION THREAD

**Discussion by Kale-Ko** [community/community#24696] [Apps, API and Webhooks]
<p>Hi, I am trying to find a way to get a users pins with the GitHub REST API, but I can’t seem to find a way.<br>
I have tried looking at /user/{username} (Didn’t contain pins data or a URL to any) and /user/{username}/pins (404)<br>
Is there something I am missing?</p>

**Comment by byrneh** [comment, marked as answer]
<p>Hi <a class="mention" href="/kale-ko">@Kale-Ko</a></p>
<p>The GraphQL API usery query has field pinnedItems if its a list of repositories and gists this profile owner has pinned to their profile.</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/graphql/reference/objects#user">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/graphql/reference/objects#user" target="_blank" rel="noopener nofollow ugc">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/39440862/181086291-f0b97afe-10d6-4f08-9b7e-6636a70b1dd9.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/graphql/reference/objects#user" target="_blank" rel="noopener nofollow ugc">Objects - GitHub Docs</a></h3>



  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
